.github/workflows/pkgdown.yaml

```diff
@@ -45,7 +53,37 @@ jobs:
         run: R CMD INSTALL .
 
       - name: Deploy package
+        if: contains(env.isPush, 'true')
         run: |
           git config --local user.email "actions@github.com"
           git config --local user.name "GitHub Actions"
           Rscript -e 'pkgdown::deploy_to_branch(new_process = FALSE)'
+
+      - name: Create website and another page
+        if: contains(env.isPush, 'false')
+        run: |
+          Rscript -e 'pkgdown::build_site()'
+          Rscript -e 'pkgdown::build_home(override=list(destination="cosmo", template = list(params=list( bootswatch = "cosmo"))))'
```

Comment:
I wanted to quickly see a Bootswatch theme still works, but maybe not needed here.

Reply:
Yeah, lets test that separately